packages/core/src/utils-hoist/dsn.ts

```diff
@@ -114,6 +117,21 @@ function validateDsn(dsn: DsnComponents): boolean {
   return true;
 }
 
+/**
+ * Extract the org ID from a DSN host.
+ *
+ * @param host The host from a DSN
+ * @returns The org ID if found, undefined otherwise
+ */
+export function extractOrgIdFromDsnHost(host: string): string | undefined {
+  const match = host.match(ORG_ID_REGEX);
+
+  if (match?.[1]) {
+    return match[1];
+  }
+  return undefined;
```

Comment:
```suggestion
  return match?.[1];
```

should be enough I think? :D